eng/pipelines/templates/jobs/ci.yml

```diff
@@ -0,0 +1,113 @@
+parameters:
+  - name: ServiceDirectory
+    type: string
+    default: ""
+  - name: BuildTargetingString
+    type: string
+    default: "azure-*"
```

Comment:
This looks to be copied from python. Do the crates follow a similar scheme?

Reply:
@heaths, it's used for package targeting in the build.   It get overridden in PRs with a package list that's based on the discovery in Language-Settings.ps1 and the pr diff.  In a manual or CI run where there's no changeset driving the package selection, we can still use this to filter packages

Reply:
I've changed the default to `*`

Reply:
After talking to @weshaggard, I've removed all of the Targeting string related code from the pipelines.   The pipelines will use the contents of the package info folder to decide what to build.